kong/target_service.go

```diff
@@ -22,6 +22,8 @@ type AbstractTargetService interface {
 	// MarkUnhealthy marks target belonging to upstreamNameOrID as unhealthy in
 	// Kong's load balancer.
 	MarkUnhealthy(ctx context.Context, upstreamNameOrID *string, target *Target) error
+	// Update updates a Target in Kong under upstreamID.
```

Comment:
Let's make sure to add a note that `PATCH` will be used for the update operation.